vizro-core/tests/unit/vizro/models/_navigation/test_nav_item.py

```diff
@@ -31,6 +31,14 @@ def test_nav_link_mandatory_and_optional(self, pages_as_list):
         assert nav_link.icon == "home"
         assert nav_link.pages == pages_as_list
 
+    @pytest.mark.parametrize("icon", ["Bar Chart", "bar chart", "bar_chart", "Bar_Chart"])
```

Comment:
When tested locally "Bar Chart" and "bar chart" are not valid icon names as they do not produce an icon.

Reply:
Can you double-check if the latest commits have been in your local branch? I've just double-checked again and it does reproduce the icon. Given that this unit test passes, the validator seems to be producing the right results as well?

Try with these configurations for example:

```
dashboard = vm.Dashboard(
    pages=[
        create_home_page(),
        create_variable_analysis(),
        create_relation_analysis(),
        create_continent_summary(),
        create_country_analysis(),
    ],
    navigation=vm.Navigation(
        nav_selector=vm.NavBar(
            items=[
                vm.NavLink(
                    label="Link 1",
                    icon="Bar Chart",
                    pages={
                        "Analysis": ["Homepage", "Variable Analysis", "Relationship Analysis", "Country Analysis"],
                    },
                ),
                vm.NavLink(
                    label="Link 2",
                    icon="Arrow Drop Down",
                    pages={
                        "Summary": ["Continent Summary"],
                    },
                ),
            ]
        )
    ),
)
```